snap/hooks/post-refresh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -eux
+
+OLD_SNAP_DATA=$(cat "${SNAP_COMMON}"/on_refresh_snap)
+rm "${SNAP_COMMON}"/on_refresh_snap
+
+for i in $(find "${SNAP_DATA}"/etc/opensearch -type f); do
+        # Some files may have been created by snap_daemon and then, will cause permission denied
+	if sed "s@${OLD_SNAP_DATA}@${SNAP_DATA}@g" $i > "$SNAP_DATA"/temp_sed_file; then
+		cat "$SNAP_DATA"/temp_sed_file > $i
+	fi
```

Comment:
That causes an awkward error with snap... Because the ```sed -i``` needs a temp file to be created (and we cannot define its path), so it fails with permission denied.

What I am doing to work around this issue is manually setting the temp file myself and, if there is a change, then copy it over $i.